### DISCUSSION TITLE
Instant View

### DISCUSSION THREAD

**Discussion by rzasharifi** [spacecowboy/Feeder#364] [Ideas]
Hi. In Telegram, there's a feature called Instant view that it make possible to see the article's page without leaving Telegram. I'm wondering if there is any chance that you could add this feature to the App?

More about it:
https://telegram.org/blog/star-giveaways-iv-in-browser
https://telegram.org/evolution#november-2016
https://telegram.org/blog/w3-browser-mini-app-store#telegram-browser
https://github.com/mozilla/readability

**Comment by spacecowboy** [comment]
The equivalent feature is "Open with CustomTab".
